### PR TITLE
CDEvent should maintain strong references to its properties.

### PR DESCRIPTION
**Pull request by uasi**
Unretained property is source of headaches:

``` objc
- (void)foo {
    self.events = [[CDEvents alloc] initWithURLs:URLs
                                           block:^(CDEvents *watcher, CDEvent *event) {
                                               self.lastEvent = event;
                                           }];
}

- (void)bar {
    // Remember the URL property is unsafe_unretained.
    NSLog(@"%@", self.lastEvent.URL); // BOOM!
}
```

### CODE REVIEW

File: CDEvent.h [hunk 1]
```diff
@@ -89,7 +89,7 @@
  *
  * @since 1.0.0
  */
-@property (unsafe_unretained, readonly) NSDate	*date;
+@property (readonly) NSDate	*date;
```
Comment (rastersize) [inline CDEvent.h]: `@property (readonly) NSDate *date;` is the same as `@property (unsafe_unretained, readonly) NSDate *date;`. Could you add `strong` to the property.

File: CDEvent.h [hunk 2]
```diff
@@ -98,7 +98,7 @@
  *
  * @since 1.0.0
  */
-@property (unsafe_unretained, readonly) NSURL	*URL;
+@property (readonly) NSURL	*URL;
```
Comment (rastersize) [inline CDEvent.h]: `@property (readonly) NSDate *date;` is the same as `@property (unsafe_unretained, readonly) NSDate *date;`. Could you add `strong` to the property.

**Comment by rastersize** [comment]
Defiantly a bug, if you fix my comments I’ll merge this :star2:

**Comment by uasi** [comment]
@rastersize That's true _for MRC_. When ARC is enabled, `@property (readonly) NSSomething *` is equivalent to `@property (strong, readonly) NSSomething *` [as stated in the Clang doc](http://clang.llvm.org/docs/AutomaticReferenceCounting.html#property-declarations).
Having said that, `@property (readonly)` is ill-formed on not-quite-recent LLVM (<3.1, that is, Xcode <4.3) with ARC enabled. I'm very willing to fix that.

**Comment by rastersize** [comment]
@uasi I see, well thank you for teaching me something new and also fixing the bug.

**Pull request merged**